internal/uvm/create_lcow.go

```diff
@@ -22,9 +22,18 @@ import (
 type PreferredRootFSType int
 
 const (
-	PreferredRootFSTypeInitRd PreferredRootFSType = 0
-	PreferredRootFSTypeVHD    PreferredRootFSType = 1
+	PreferredRootFSTypeInitRd PreferredRootFSType = iota
+	PreferredRootFSTypeVHD
+)
+
+type OutputHandlingType int
+
+const (
+	OutputHandlingTypeLogrus OutputHandlingType = iota
```

Comment:
What is the difference between these two?

Reply:
I'll add a comment explaining. OutputHandlingTypeLogrus means we parse the output as if it were a Logrus message (this is the existing behavior in master, used to receive messages from GCS). OutputHandlingTypeStdout means we just print it to stdout (e.g. to display dmesg).